README.md

```diff
@@ -79,17 +74,15 @@ Or see the [installation guide](https://wxt.dev/guide/installation.html) to get
 
 WXT is a [MIT-licensed](https://github.com/wxt-dev/wxt/blob/main/LICENSE) open source project with its ongoing development made possible entirely by the support of these awesome backers. If you'd like to join them, please consider [sponsoring WXT's development](https://github.com/sponsors/wxt-dev).
 
-<a href="https://github.com/sponsors/wxt-dev"><img alt="WXT Sponsors" src="https://raw.githubusercontent.com/wxt-dev/static/refs/heads/main/sponsorkit/sponsors.svg"></a>
+[![WXT Sponsors](https://raw.githubusercontent.com/wxt-dev/static/refs/heads/main/sponsorkit/sponsors.svg)](https://github.com/sponsors/wxt-dev)
 
 ## Contributors
 
 <!-- automd:contributors author="aklinker1" license="MIT" github="wxt-dev/wxt" -->
 
 Published under the [MIT](https://github.com/wxt-dev/wxt/blob/main/LICENSE) license.
 Made by [@aklinker1](https://github.com/aklinker1) and [community](https://github.com/wxt-dev/wxt/graphs/contributors) 💛
-<br><br>
-<a href="https://github.com/wxt-dev/wxt/graphs/contributors">
-<img src="https://contrib.rocks/image?repo=wxt-dev/wxt" />
-</a>
+
+[![Contributors](https://contrib.rocks/image?repo=wxt-dev/wxt)](https://github.com/wxt-dev/wxt/graphs/contributors)
```

Comment:
Let's ignore this change as well, it's generated by auto-md, so it will be reverted the next time that tool is ran.

Reply:
Reverted.